backend/de.metas.elasticsearch/src/main/java/de/metas/fulltextsearch/config/ESQueryTemplate.java

```diff
@@ -63,6 +63,7 @@ public String resolve(@NonNull final Evaluatee evalCtx)
 	{
 		final ToJsonEvaluatee evalCtxEffective = new ToJsonEvaluatee(evalCtx);
 		evalCtxEffective.skipConvertingToJson(PARAM_orgFilter.getName());
+		evalCtxEffective.skipConvertingToJson(PARAM_query.getName());
```

Comment:
why?

Reply:
I put this on the skip list because otherwise it would come with a double set of quotes from here: 
```
  "match": {
            "poreference": {
              "query": "@query@",  
```

Reply:
I remember trying it and it not working. Not that I think back, I might have been biased because of the ES_QueryCommand definition I used and the fix seemed to fix my problem. So this code change could be useless if the command is rewritten.

Reply:
i would prefer that because if not, not sure how searching by something like `aa " bb` would work